app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+# Task Controller
+class TasksController < ApplicationController
+  load_and_authorize_resource find_by: :sequence_num
+
+  def index
+    @tasks = Task.all
+    @users = User.all
+    if @tasks.nil?
+      flash[:notice] = 'No Task exist yet'
+      redirect_to new_task_path
+    end
+  end
+
+  def show
+    @users = User.all
+  end
+
+  def new
+    @users = User.all
+  end
+
+  def edit
+    @users = User.all
+  end
+
+  def create
+    @task.project_id = 2
+    if @task.save
+      flash[:notice] = 'Task created successfully'
+      render :show
+    else
+      errors = ''
+      @task.errors.full_messages.each do |msg|
```

Comment:
we can use join function on array
`errors = @task.errors.full_messages.join(', ')`